projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/usage-bar/index.tsx

```diff
@@ -49,17 +50,37 @@ function UsageControl( {
 	hasFeature,
 	requestsCount,
 	requestsLimit,
-}: Pick< AIFeatureProps, 'isOverLimit' | 'hasFeature' | 'requestsCount' | 'requestsLimit' > ) {
+	usagePeriod,
+}: Pick<
+	AIFeatureProps,
+	'isOverLimit' | 'hasFeature' | 'requestsCount' | 'requestsLimit' | 'usagePeriod'
+> ) {
+	// Compute the number of days from start to now
+	let resetMsg = '';
+	if ( usagePeriod?.currentStart ) {
+		const nextPeriodStart = new Date( usagePeriod.nextStart );
+
+		// Number of days in the current period
+		const numberOfDays = Math.floor(
```

Comment:
```suggestion
		const numberOfDays = Math.ceil(
```

Should we use `Math.ceil` in this case?

Consider for example that the reset date is `2023 Oct 31, 00:00:00` and the time now is `2023 Oct 30, 19:25:00`. 

By using  `floor`, we get `0` as the `numberOfDays`, which is odd. If it was `0`, it should be reset already. In this example, maybe `1` would be more appropriate (meaning "will reset tomorrow").

What do you think?

Reply:
I think you're right. It's more beneficial customers.